mautic/mautic
# Fix webhook utf8 string issue

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Webhook endpoint returning some invalid character in response so it is not able to precess, getting mysql error:

> `An exception occurred while executing 'INSERT INTO mautic_webhook_logs (status_code, date_added, note, runtime, webhook_id) VALUES (?, ?, ?, ?, ?)' with params [200, "2021-04-19 07:05:45", "\x7b\x22\x6d\x61\x75\x74\x69\x63\x2e\x66\x6f\x72\x6d\x5f\x6f\x6e\x5f\x73\x75\x62\x6d\x69\x74\x22\x3a\x5b\x7b\x22\x70\x61\x79\x6c\x6f\x61\x64\x22\x3a\x7b\x22\x70\x61\x79\x6c\x6f\x61\x64\x22\x3a\x7b\x22\x73\x75\x62\x6d\x69\x73\x73\x69\x6f\x6e\x22\x3a\x7b\x22\x69\x64\x22\x3a\x32\x35\x31\x33\x38\x2c\x22\x69\x70\x41\x64\x64\x72\x65\x73\x73\x22\x3a\x7b\x22\x69\x70\x41\x64\x64\x72\x65\x73\x73\x22\x3a\x22\x31\x38\x35\x2e\x36\x38\x2e\x37\x38\x2e\x32\x34\x34\x22\x2c\x22\x69\x64\x22\x3a\x33\x39\x35\x39\x38\x34\x31\x2c\x22\x69\x70\x44\x65\x74\x61\x69\x6c\x73\x22\x3a\x7b\x22\x63\x69\x74\x79\x22\x3a\x22\x4c\x65\x75\x74\x65\x6e\x62\x61\x63\x68\x22\x2c\x22\x72\x65\x67\x69\x6f\x6e\x22\x3a\x22\x42\x61\x64\x65\x6e\x2d\x57\xfc\x72\x74\x74\x65\x6d\x62\x65\x72\x67\x22\x2c", 0.08, 13]:
> 
> SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xFCrttem...' for column 'note' at row 1`

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create webhook with some events (Ex. Send email)
2. now send email for any contact
3. Now webhook will be process
4. if webhook response body contains invalid char like binary then it will show database query error.

#### Steps to test this PR:
1. Create webhook with some events (Ex. Contact Points Changed Event)
2. now change point for any contact
3. Now webhook will be process
4. if webhook response body contains  invalid char like binary then it should not throw any db error.


#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
